src/EFCore.PG/Storage/Internal/NpgsqlTypeMappingSource.cs

```diff
@@ -341,6 +343,7 @@ public NpgsqlTypeMappingSource(
             { typeof(NpgsqlPath), _path },
             { typeof(NpgsqlPolygon), _polygon },
             { typeof(NpgsqlCircle), _circle },
+            { typeof(NpgsqlCube), _cube },
```

Comment:
I think you want to add an entry to storeTypeMappings above as well, that would enable scaffolding of cube properties from existing databases.